OpenRA.Mods.Common/Activities/Transform.cs

```diff
@@ -71,15 +68,10 @@ public override bool Tick(Actor self)
 				return false;
```

Comment:
This can be `return true` now, right?

Reply:
True :stuck_out_tongue: 

But it doesn't matter either way. The childactivity will never finish so this will never actually proceed to the next activity.